esileapclient/common/utils.py

```diff
@@ -0,0 +1,44 @@
+import re
+import operator
+
+
+def parse_property_filter(filter_str):
+    """Parse a property filter string into a key, operator, and value."""
+    match = re.match(r'([^><=]+)([><=]+)(.+)', filter_str)
+    if not match:
+        raise ValueError(f"Invalid property filter format: {filter_str}")
+    key, op_str, value_str = match.groups()
+    ops = {
+        '>=': operator.ge,
+        '<=': operator.le,
+        '>': operator.gt,
+        '<': operator.lt,
+        '=': operator.eq,
+    }
```

Comment:
I'd pull `ops` out of the function and make it a constant

Reply:
Done.